### PR TITLE
fix(package): add domhandler to dependencies

### LINKED ISSUE #231: Module '"../@types/domhandler"' has no exported member
## Expected Behavior

Run OK

## Actual Behavior

Run KO

## Steps to Reproduce

When used with TypeScript (4.1.5), it's suggest to install `@types/domhandler` and `@types/htmlparser2`.

It's fine. But when it's done, I've got these errors :

image

## Environment

- Version: 1.2.4
- Platform: macOS
- Browser: Chrome

### PR DESCRIPTION
**Pull request by kfritsch**
added missing dependency for "domhandler": "^4.0.0"

fixes #231

## Checklist:

- [ ] Tests
- [ ] Documentation
- [ ] Types

Explanation can be found in the issue. Only tested the workaround explained in the issue.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
     "dom"
   ],
   "dependencies": {
+    "domhandler": "^4.0.0",
```
Comment (remarkablemark) [inline package.json:40]: Do you mind making this an exact dependency? `4.0.0`

**Review by remarkablemark — CHANGES_REQUESTED** [review]
Could you amend your commit message so it's more descriptive?

```sh
git commit --amend -m 'fix(package): add domhandler to dependencies' -m 'Fixes #240'
git push -f
```

File: package.json (hunk 1 above)
Comment (kfritsch) [reply package.json:40]: not at all

**Review by remarkablemark — APPROVED** [review]
LGTM, thanks for opening this PR @kfritsch

**Comment by remarkablemark** [comment]
@kfritsch I'll make a release later tonight

**Comment by remarkablemark** [comment]
Published [v1.2.5](https://github.com/remarkablemark/html-react-parser/releases/tag/v1.2.5):

[npm](https://www.npmjs.com/package/html-react-parser):

```sh
npm i html-react-parser@1.2.5
```

[Yarn](https://yarnpkg.com/package/html-react-parser):

```sh
yarn add html-react-parser@1.2.5
```

**Pull request merged**